src/components/shared/misc/ImageUploader.vue

```diff
@@ -129,7 +137,7 @@
             return {
                 src: this.defaultSrc,
                 loading: false,
-                showModal: false,
```

Comment:
I think this should remain part of the state for this component.
If we want to open and close the modal from the parent we can do something like this...
```js
this.$refs.uploader.showModal = true 
``` 
Either way, this is a breaking change that will need to be reflected in the docs.